docs/css-prop.md

```diff
@@ -32,27 +82,37 @@ render(
 )
 ```
 
-You can also pass objects directly to the css prop, for more usage with objects, look at [the object styles page](/docs/object-styles.md).
+## Style Precedence
 
-```jsx
-// @live
+The `className` prop value overrides the styles provided to the `css` prop which is an important distinction when passing class names generated by emotion. Emotion removes generated class names from the `className` prop value, their underlying styles composed on top of the styles supplied to the `css` prop generating a new, single class name. The precedence order may seem counter-intuitive, but it allows developers predictably compose components and their styles.
+
+The following example demonstrates a potential problem if the `css` prop overwrites styles from `className` prop.
```

Comment:
This is not very clear. Maybe just say, if the same style is defined in the css prop *and* className, the style of the classname will be used (has a higher precedence).

Reply:
Yeah, this is rough. Thanks.